shell/src/main/java/alluxio/cli/fsadmin/report/CapacityCommand.java

```diff
@@ -58,7 +59,7 @@
   public CapacityCommand(BlockMasterClient blockMasterClient, PrintStream printStream) {
     mBlockMasterClient = blockMasterClient;
     mPrintStream = printStream;
-    mStringBuilder = new StringBuilder();
+    initValues();
```

Comment:
I put the initVariable method in the collectWorkerInfo method because it will init the member variables that we want to save worker info to. This way may be more reasonable. 

Put in the constructor is not a good choice when those variables do not need the constructor inputs.

Thanks for pointing out that.